doc/array/splice.md

```diff
@@ -1,2 +1,46 @@
 @property {function} can-observe/splice splice
 @parent can-observe/array
+
+@description Insert and remove elements from an observe array.
+@signature `list.splice(index[, howMany[, ...newItems]])`
+
+  `splice` inserts and removes elements from an observe array.
+
+  ```js
+  import { observe } from "can/everything";
+
+  const names = new observe.Array(['Alice', 'Bob', 'Chris']);
+  console.log(names.splice(0, 1)); //-> ['Alice']
+
+  console.log(names); //-> ['Bob', 'Chris']
+  ```
+  @codepen
+
+  @param {Number} Where to start removing or inserting elements
+
+  @param {Number} The number of elements to remove, if _howMany_ is not provided `splice` will remove all elements from the `index` to the end of the array.
+
+  @param {*} elements Items to insert into the array
```

Comment:
I think `elements` should be `newItems`

Also, the other params don’t actually name the param, i.e. it should be `@param {Number} index Where to…`

Reply:
Good spot. 👍